Lib/operator.py

```diff
@@ -415,7 +415,7 @@ def ixor(a, b):
 except ImportError:
     pass
 else:
-    from _operator import __doc__
+    from _operator import __doc__  # noqa
```

Comment:
Considering that `__doc__` has a specific meaning, it feels like this is a bug in whatever linter you are using.

Reply:
Not really, an unused-import check shouldn't complicate itself by making exceptions for special names as such cases are rare.

Reply:
Well I mean the majority of the time that `__doc__` and `__version__` are imported, it is to expose them in a new namespace. I can't imagine that these are commonly *used*, but they also are not the kind of thing that you add to `__all__`. Seems to me that linters shouldn't be warning in the common, correct case.